src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -1,227 +1,239 @@
-import React, { Component } from 'react';
+import React, { useEffect, useState } from 'react';
 import IMG from 'react-graceful-image';
 import { Typography, Grid, Divider } from '@material-ui/core';
 import PropTypes from 'prop-types';
-import user from 'services/user';
 import { Link } from 'react-router-dom';
+import userService from 'services/user';
+import handleViewport from 'react-in-viewport';
 
 import './peopleSearchResult.css';
 
-export default class PeopleSearchResult extends Component {
-  constructor(props) {
-    super(props);
+/*Const string was created with https://png-pixel.com/ .
+ *It is a 1 x 1 pixel with the same color as gordonColors.neutral.lightGray (7/9/21)
+ *Although this doesn't use the gordonColors themes directly,
+ *the end result is much cleaner and faster than using the placeholderColor tag of react-graceful-image.
+ */
+const GORDONCOLORS_NEUTRAL_LIGHTGRAY_1X1 =
```

Comment:
Giving `<IMG>` tags an image to serve as a placeholder was far better than using `null` and using the built-in placeholder feature of `react-graceful-image`; in fact, one reason why the search result images didn't load until they were all loaded was because of this use of `null`.

Reply:
The only thing I don't like about this is that it's not bound to the gordon colors in `theme`. Hypothetically, if `gordonColors.neutral.lightGray` ever changed, we'd manually change this. The only way I could think of to make this bound was to have a function that took in a color and made a 1x1 base64 pic of that color, but I couldn't figure that out.